src/udiskslinuxencrypted.c

```diff
@@ -129,6 +129,9 @@ void
 udisks_linux_encrypted_update (UDisksLinuxEncrypted   *encrypted,
                                UDisksLinuxBlockObject *object)
 {
+  // All objects with the Encrypted interface are LUKS.
```

Comment:
Please add a comment that this will need to be updated soon for *LUKS2* right away. We probably need a new function in libblockdev to determine the real metadata size instead of just using a constant.

Reply:
Done.